charts/rstudio-connect/README.md

```diff
@@ -1,6 +1,6 @@
 # Posit Connect
 
-![Version: 0.7.6](https://img.shields.io/badge/Version-0.7.6-informational?style=flat-square) ![AppVersion: 2024.08.0](https://img.shields.io/badge/AppVersion-2024.08.0-informational?style=flat-square)
+![Version: 0.7.7](https://img.shields.io/badge/Version-0.7.7-informational?style=flat-square) ![AppVersion: 2024.08.0](https://img.shields.io/badge/AppVersion-2024.08.0-informational?style=flat-square)
```

Comment:
Why is this not `0.7.8`? I would expect this to have the latest version.

Reply:
It should, there was a bit of messiness with the merge, Im trying to work through that

Reply:
I don't understand why the diff here isn't showing main but the diff from the original commits. I used vscode to do the merge and am not liking the result. Debating blowing this away and getting a clean Branch & PR